src/commands/remote_signer_setup.py

```diff
@@ -220,3 +268,115 @@ async def main(remove_existing_keys: bool) -> None:
+                private_keys.append(int.from_bytes(keystore, 'big'))
+            except (json.JSONDecodeError, KeyError) as e:
+                click.secho(
+                    f'Failed to load keystore {filename}. Error: {str(e)}.',
+                    fg='red',
+                )
+
+    if key_shares is None or 'keystores' not in key_shares or 'passwords' not in key_shares:
+        raise ValueError('key_shares is malformed or None')
+
+    with click.progressbar(
+        length=len(key_shares['keystores']),
+        label='Loading key shares...\t\t',
+        show_percent=False,
+        show_pos=True,
+    ) as progressbar:
+        for index, key_share_str in enumerate(key_shares['keystores']):
+            try:
+                keystore_password = key_shares['passwords'][index]
+                # Parse key_share from string to dictionary
+                key_share = json.loads(key_share_str)
+                keystore = ScryptKeystore.from_json(key_share).decrypt(keystore_password)
+                private_keys.append(int.from_bytes(keystore, 'big'))
+            except (json.JSONDecodeError, KeyError) as e:
+                click.secho(
+                    f'Failed to load keystore {key_share_str}. Error: {str(e)}.',
+                    fg='red',
+                )
+            progressbar.update(1)
+
+    database = Database(
+        db_url=str(db_url),
+    )
+    encryptor = Encryptor(db_encrypt_key)
+
+    database_records = _encrypt_private_keys(
+        private_keys=private_keys,
+        encryptor=encryptor,
+    )
+    database.upload_keys(keys=database_records)
+    total_keys_count = database.fetch_public_keys_count()
+
+    configs = [
+        _read_config_file_to_record(
+            settings.remote_signer_config_file, 'remote_signer_config.json'
+        ),
+        _read_config_file_to_record(settings.deposit_data_file, 'deposit_data.json'),
+    ]
+    database.upload_configs(configs)
+
+    click.clear()
+
+    click.secho(
+        f'The database contains {total_keys_count} validator keys.\n'
+        f"The decryption key: '{encryptor.str_key}'\n"
+        'The configuration files have been uploaded to the remote database.',
+        bold=True,
+        fg='green',
+    )
+
+
+def _read_config_file_to_record(filepath: Path, filename: str) -> DatabaseConfigRecord:
+    """Reads a JSON config file and returns a DatabaseConfigRecord instance."""
+    with open(filepath, encoding='utf-8') as file:
+        data = json.dumps(json.load(file))
+    return DatabaseConfigRecord(name=filename, data=data)
+
+
+def _encrypt_private_keys(private_keys: list[int], encryptor: Encryptor) -> list[DatabaseKeyRecord]:
+    """
+    Returns prepared database key records from the private keys.
+    """
+
+    click.secho('Encrypting database keys...', bold=True)
+    key_records: list[DatabaseKeyRecord] = []
+    for private_key in private_keys:
+        encrypted_private_key, nonce = encryptor.encrypt(str(private_key))
```

Comment:
It's better to convert private key to bytes and encrypt bytes rather than int to str to bytes